src/Sdk/RSWebApi/RunServiceHttpClient.cs

```diff
@@ -64,7 +64,7 @@ public class RunServiceHttpClient : RawHttpClientBase
             HttpMethod httpMethod = new HttpMethod("POST");
             var payload = new AcquireJobRequest
             {
-                StreamID = messageId
+                JobMessageID = messageId
```

Comment:
will this cause any compat break on the server side, feels like either we accept the break or send both fields in a short period of time so we can release the runner ahead of the server.

Reply:
Yeah, the easiest way is to release the runner ahead of the server and send both fields for a short period of time.

The corresponding server change is here:  https://github.com/github/actions-run-service/pull/525/files#diff-9a30818e05fad1a13d0fcde9639c0ac3ac29a7902e95eeba19659808b2192b45

Reply:
Do you want to include that in the PR? sending both `JobMessageId` and `StreamId`

Reply:
Both fields are sent.